mettle/worker/worker.go

```diff
@@ -715,28 +715,32 @@ func (w *worker) execute(req *pb.ExecuteRequest, action *pb.Action, command *pb.
 }
 
 func containsAllOutputPaths(cmd *pb.Command, ar *pb.ActionResult) bool {
-	paths := make(map[string]struct{}, len(ar.OutputDirectories)+len(ar.OutputFiles)+len(ar.OutputSymlinks)+len(ar.OutputDirectorySymlinks))
+	paths := make(map[string]bool, len(ar.OutputDirectories)+len(ar.OutputFiles)+len(ar.OutputSymlinks)+len(ar.OutputDirectorySymlinks)+len(ar.OutputFileSymlinks))
```

Comment:
Don't think it really matters here but the empty struct is more efficient - it takes no space whereas the bool will need a byte (or probably more for alignment)

I also tend to prefer it to be clear that it has no meaning. Do agree the check is a bit simpler though. Guess maybe we'll get a first class set with generics!